src/schematic-view.c

```diff
@@ -588,7 +588,7 @@ static void deselect_all_cmd (GtkWidget *widget, SchematicView *sv)
 
 static void delete_cmd (GtkWidget *widget, SchematicView *sv)
 {
-	sheet_delete_selection (sv->priv->sheet);
+	sheet_delete_selection (sv->priv->sheet, TRUE);
 }
```

Comment:
Can you please make this an enum? I find it very hard to read when TRUE and FALSE start to appear all over the place.